Formula/llvm.rb

```diff
@@ -25,6 +25,12 @@ class Llvm < Formula
       sha256 "62014ddad6d5c485ecedafe3277fe7978f3f61c940976e3e642536726abaeb68"
     end
 
+    # Fix for D89156 that broke lldb build on MacOS
+    patch do
+      url "https://raw.githubusercontent.com/abpostelnicu/formula-patches/25e93dc16dd1c9a1acd1704f7ad78955a11f2a26/llvm/lldb_build_fix.patch"
```

Comment:
```suggestion
      url "https://raw.githubusercontent.com/Homebrew/formula-patches/25e93dc16dd1c9a1acd1704f7ad78955a11f2a26/llvm/lldb_build_fix.patch"
```
Where did this patch come from? Can you also leave a comment about when it can be removed?

Reply:
I'm building this patch locally and updated the corresponding phabricator commit. I will also push the patch to fabricator. When everything is finished I will change the link with the one from Homebrew/formula-patches#348

Reply:
If you're pushing it to phabricator, then we should try to use a patch from there if we can.

Reply:
@carlocab https://reviews.llvm.org/D95683

Reply:
Will the diff at https://reviews.llvm.org/file/data/ud3rpyci65brpi32praa/PHID-FILE-cdiialsygwx24dl7twob/D95683.diff apply?

Reply:
> Will the diff at https://reviews.llvm.org/file/data/ud3rpyci65brpi32praa/PHID-FILE-cdiialsygwx24dl7twob/D95683.diff apply?

Done the modification but now we have an external dependency on the llvm phabricator repo that we cannot control. I would have preferred to have Homebrew/formula-patches#348 merged and crete the dependency against that, something that we can control. But anyway I'm fine with either solution as long as we get this merged.